Modules/Devices/src/Devices.Infrastructure/PushNotifications/DirectPush/DirectPnsCommunicationOptions.cs

```diff
@@ -37,7 +37,7 @@ public bool HasConfigForAppId(string appId)
             return serviceAccountForServiceAccountNameExists;
         }
 
-        public string? GetServiceAccountForAppId(string appId)
```

Comment:
Keep it. 

In other words: ignore the warning that says that you cannot use the `?` annotation in a non-nullable context